Tella/Scenes/Uwazi/ViewModel/UwaziViewModel.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  UwaziViewModel.swift
+//  Tella
+//
+//  Created by Gustavo on 25/09/2023.
+//  Copyright © 2023 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+
+
+class UwaziViewModel: ObservableObject {
+    
```

Comment:
I created this `UwaziViewModel` to handle the general pageView logic of uwazi. That way we have the `UwaziTemplateViewModel` focusing only on template stuff. 

```
@Published var draftEntities : [Report] = []
    @Published var outboxedEntities : [Report] = []
    @Published var submittedEntities : [Report] = []
```

this should be type `UwaziEntity` but we have that type in another PR. I suggest we merge this with that type and then on the entity one I will change it with the correct one.